src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.common;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+
+public class UtilsTest {
+    
+    /**
+     * Test Utils isAnyNull method
+     */
+    
+    @Test
+    public void IsAnyNull_isTrue() {
+        Object[] testData1_true = {null,null,null,null,null};
+        assertTrue("Return true when all are nulls",Utils.isAnyNull(testData1_true));       
+    }
+    
+    @Test
+    public void IsAnyNull_isFalse(){
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.